README.rst

```diff
@@ -1,41 +1,230 @@
+Energy isLANDs, project homepage `HERE <https://elandh2020.eu/>`__).
+A graphical user interface for the MVS will be integrated.
 
-*Latest release*
-Check the [latest release](https://github.com/rl-institut/multi-vector-simulator/releases/latest). Please check the [CHANGELOG.md](https://github.com/rl-institut/multi-vector-simulator/blob/master/CHANGELOG.md) for past updates and changes.
+*Latest release*: Check the `latest release <https://github.com/rl-institut/multi-vector-simulator/releases/latest>`__.
+Please check the `CHANGELOG.md <https://github.com/rl-institut/multi-vector-simulator/blob/master/CHANGELOG.md>`__ for past updates and changes.
 
-*Disclaimer*
-As the MVS is still under development, changes might still occur in the code as well as code
- structure. If you want to try the MVS, please make sure to check this project regularly.
+*Disclaimer*: As the MVS is still under development, changes might still occur in the code as well as code structure.
+If you want to try the MVS, please make sure to check this project regularly.
 
-For advanced programmers: You can also use the `dev` branch that includes the latest updates and
- changes. You find the changelog [HERE](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md).
+For advanced programmers: You can also use the ``dev`` branch that includes the latest updates and changes.
+You find the changelog `HERE <https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md>`__.
 
-# Getting started
+Getting started
+===============
 
-If you are interested to try out the code, please feel free to do so! In case that you are planning to use it for a specific or a larger-scale project, we would be very happy if you would get in contact with us, eg. via issue. Maybe you have ideas that can help the MVS move forward? Maybe you noticed a bug that we can resolve?
+If you are interested to try out the code, please feel free to do so! In case that you are planning to use it for a specific or a larger-scale
+project, we would be very happy if you would get in contact with us, eg. via creating a github issue.
+Maybe you have ideas that can help the MVS move forward? Maybe you noticed a bug that we can resolve?
 
-We are still working on including a readthedocs for the MVS. Some information on this tool and code is already available [here](https://multi-vector-simulator.readthedocs.io/en/stable/) (stable version, latest developments [here](https://multi-vector-simulator.readthedocs.io/en/latest/)).
+We are still working on including a readthedocs for the MVS. Some information on this tool and code is already
+available `here <https://multi-vector-simulator.readthedocs.io/en/stable/>`__ (stable version,
+latest developments `here <https://multi-vector-simulator.readthedocs.io/en/latest/>`__).
 
-## Setup and installation
+Setup and installation
+----------------------
 
-.. include:: docs/Installation.rst
```

Comment:
Actually, we should keep `.. include:: docs/Installation.rst`. The sense of this is, that we no longe have two places to update the Installation instructions (Readme and installation.rst), but only one.